src/features/flows/nRF52/index.ts

```diff
@@ -44,7 +44,7 @@ const programConfig = [
         name: 'Peripheral LED Button Service',
         type: 'jlink',
         description:
-            'Check that the LEDs and buttons on the DK are working with this sample.',
+            'This sample demonstrates how to use the LED Button Service (LBS).',
```

Comment:
My thoughts evolved since I sent my mail in reply to Ingar's thread. Let's not promise working with BLE off the bat, but prepare the user for what he can do later:
```suggestion
            'Program the LED Button Service (LBS) to test it later with Bluetooth(r) LE.',
```